app/components/header/header-navigation.njk

```diff
@@ -10,10 +10,7 @@
         {
           "url"  : "https://www.nhs.uk/conditions",
           "label" : "Health A-Z",
-          "classes": "app-header__navigation-item--current",
-          "attributes": {
-            "aria-current": "true"
-          }
+          "classes": "app-header__navigation-item--my-modifier"
```

Comment:
Do we need this in the example?

Reply:
Probably not. Maybe we ought to have some tests to make sure that the custom `classes` and `attributes` work across all the components, but that could be a separate job.

Reply:
Removed.